Sources/StreamChatSwiftUI/ChatChannel/MessageList/Polls/PollsHelpers.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+private struct PollsBackgroundModifier: ViewModifier {
+    
+    @Injected(\.colors) var colors
+    
+    func body(content: Content) -> some View {
+        content
+            .padding()
+            .background(Color(colors.background1))
+            .cornerRadius(16)
+    }
+}
+
+extension View {
+    func withPollsBackground() -> some View {
+        modifier(PollsBackgroundModifier())
+    }
+}
+
+struct PollDateIndicatorView: View {
+    
+    @Injected(\.fonts) var fonts
+    @Injected(\.utils) var utils
+    @Injected(\.colors) var colors
+    
+    var dateFormatter: (Date) -> String {
+        utils.pollsDateFormatter.dateString(for:)
+    }
+    
+    var date: Date
+    
+    var body: some View {
+        Text(dateFormatter(date))
+            .font(fonts.subheadline)
+            .foregroundColor(Color(colors.textLowEmphasis))
+    }
+}
+
+class PollsDateFormatter {
+    
+    let dateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.setLocalizedDateFormatFromTemplate("MMM dd HH:mm")
```

Comment:
@martinmitrevski I changed this to localized date format. Otherwise we do not respect the month and day order in different languages.

<img width="860" alt="old" src="https://github.com/GetStream/stream-chat-swiftui/assets/1469907/7f896988-2356-44a7-b012-1537e334893d">

<img width="841" alt="new" src="https://github.com/GetStream/stream-chat-swiftui/assets/1469907/e1745ee7-789e-4205-b30f-b1459364444a">